packages/rest/src/parser.ts

```diff
@@ -83,6 +87,18 @@ async function loadRequestBodyIfNeeded(
 
   const contentType = getContentType(request);
   debug('Loading request body with content type %j', contentType);
+
+  if (
+    contentType &&
+    contentType.startsWith('application/x-www-form-urlencoded')
```

Comment:
I was playing around today with this, and I found that there can also be `multipart/form-data`  ?

Reply:
`multipart/form-data` is usually used for file upload. See https://www.w3.org/TR/html401/interact/forms.html#h-17.13.4.

Reply:
correct, I was trying to upload a file from a form, but I find this very uncommon, as we can have a service on the client to communicate to the rest api.

Reply:
We need to add `multipart/form-data` support for sure to enable file uploads.

Reply:
The issue was reported by a user who wants to use simple html form. I think `application/x-www-form-urlencoded` is good enough for that.

Supporting `multipart/form-data` would be much more involved. We had `loopback-component-storage` to allow that in the past. We will need to use a module like https://github.com/mscdex/busboy#readme

Reply:
See https://github.com/expressjs/body-parser for more details.

Reply:
I agree 100%, thanks @raymondfeng and good luck today with your presentation!.

Reply:
FYI: handling file uploads is tracked by this GH issue: https://github.com/strongloop/loopback-next/issues/1452